src/locales/en-US/list.ftl

```diff
@@ -2,10 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-## common
+## Firefox / Lockbox Brand
+##
+## Firefox and Lockbox must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
+##
+## Reference: https://www.mozilla.org/styleguide/communications/translation/
```

Comment:
Let's remove this, sadly they killed that page right after we added the comment. Also, leave an empty line after the section comment.